File: libatsc3Module/src/main/java/org/ngbp/libatsc3/ndk/entities/held/Held.java
package org.ngbp.libatsc3.ndk.entities.held;

public class Held {
    String appContextId;
    String bcastEntryPackageUrl;
    String bcastEntryPageUrl;
    int coupledServices;
    boolean appRendering;

    public String getAppContextId() {
        return appContextId;
    }

    public String getBcastEntryPackageUrl() {
        return bcastEntryPackageUrl;
    }

    public String getBcastEntryPageUrl() {
        return bcastEntryPageUrl;
    }

    public int getCoupledServices() {
        return coupledServices;
    }

    public boolean isAppRendering() {
        return appRendering;
    }
}

File: libatsc3Module/src/main/java/org/ngbp/libatsc3/entities/held/Held.kt
package org.ngbp.libatsc3.entities.held

data class Held (
    var appContextId: String? = null,
    var bcastEntryPackageUrl: String? = null,
    var bcastEntryPageUrl: String? = null,
    var coupledServices: Int = 0,
    var appRendering: Boolean = false
)